x-pack/plugin/esql/qa/testFixtures/src/main/resources/data/dense_vector.csv

```diff
@@ -0,0 +1,3 @@
+id:l, vector:dense_vector
+0,    [1.0, 2.0, 3.0]
+1,    [4.0, 5.0, 6.0]
```

Comment:
this might be very nitpicky - but can we add another `dense_vector` value that does not have ordered values?
this might be why we did not caught https://github.com/elastic/elasticsearch/pull/126456#discussion_r2046935314 during tests

Reply:
mvOrdered does not impact retrieval as it's used as an optimization in some cases. But adding unordered data  uncovered a small fix that needed to be done to support multivalued style fields: 4b2126eeed19c6970ee60c4654eb92f75839f95f